packages/amplify-graphql-index-transformer/src/__tests__/__snapshots__/amplify-graphql-index-transformer.test.ts.snap

```diff
@@ -4023,178 +3915,6 @@ $util.toJson($UpdateItem)
 #else
   $util.toJson($ctx.result)
 #end
-## [End] Get ResponseTemplate. **",
-  "Mutation.updateContentCategory.init.1.req.vtl": "## [Start] Initialization default values. **
-$util.qr($ctx.stash.put(\\"defaultValues\\", $util.defaultIfNull($ctx.stash.defaultValues, {})))
-#set( $updatedAt = $util.time.nowISO8601() )
-$util.qr($ctx.stash.defaultValues.put(\\"updatedAt\\", $updatedAt))
-$util.toJson({
-  \\"version\\": \\"2018-05-29\\",
-  \\"payload\\": {}
-})
-## [End] Initialization default values. **",
-  "Mutation.updateContentCategory.postAuth.1.req.vtl": "## [Start] Merge default values and inputs. **
-#set( $mergedValues = $util.defaultIfNull($ctx.stash.defaultValues, {}) )
-$util.qr($mergedValues.putAll($util.defaultIfNull($ctx.args.input, {})))
-## [End] Merge default values and inputs. **
-## [Start] Validate update mutation for @index 'ContentByCategory'. **
-#set( $hasSeenSomeKeyArg = false )
-#set( $keyFieldNames = [\\"type\\", \\"language\\", \\"datetime\\"] )
-#foreach( $keyFieldName in $keyFieldNames )
-#if( $ctx.args.input.containsKey(\\"$keyFieldName\\") ) #set( $hasSeenSomeKeyArg = true ) #end
-#end
-#foreach( $keyFieldName in $keyFieldNames )
-  #if( $hasSeenSomeKeyArg && !$ctx.args.input.containsKey(\\"$keyFieldName\\") )
-    $util.error(\\"When updating any part of the composite sort key for @index 'ContentByCategory', you must provide all fields for the key. Missing key: '$keyFieldName'.\\")
-  #end
-#end
-## [End] Validate update mutation for @index 'ContentByCategory'. **
-#if( $util.isNull($ctx.stash.metadata.dynamodbNameOverrideMap) )
-  $util.qr($ctx.stash.metadata.put(\\"dynamodbNameOverrideMap\\", {
-  \\"type#language#datetime\\": \\"typeLanguageDatetime\\"
-}))
-#else
-  $util.qr($ctx.stash.metadata.dynamodbNameOverrideMap.put(\\"type#language#datetime\\", \\"typeLanguageDatetime\\"))
-#end
-#if( $hasSeenSomeKeyArg )
-  $util.qr($ctx.args.input.put(\\"type#language#datetime\\",\\"\${mergedValues.type}#\${mergedValues.language}#\${mergedValues.datetime}\\"))
-#end
-{}",
-  "Mutation.updateContentCategory.req.vtl": "## [Start] Mutation Update resolver. **
```

Comment:
Some of the things being removed here related to queries and mutations look concerning. Can you run the [`@index` E2E tests](https://github.com/aws-amplify/amplify-cli/blob/master/packages/graphql-transformers-e2e-tests/src/__tests__/IndexTransformer.e2e.test.ts) against this change (it should only take a few minutes).

Reply:
both [SearchableModelTransformerV2](https://github.com/aws-amplify/amplify-cli/blob/master/packages/graphql-transformers-e2e-tests/src/__tests__/SearchableModelTransformerV2.e2e.test.ts) and [IndexTransformer](https://github.com/aws-amplify/amplify-cli/blob/master/packages/graphql-transformers-e2e-tests/src/__tests__/IndexTransformer.e2e.test.ts) pass